code/modules/procedural mapping/loot.dm

```diff
@@ -0,0 +1,715 @@
+			/obj/item/weapon/melee/telebaton,
+			/obj/item/weapon/gun/energy,
+			/obj/item/weapon/gun/siren,
+			/obj/item/weapon/gun/siren/caduceus,
+			/obj/item/weapon/gun/siren/supersoaker,
+			/obj/item/weapon/gun/portalgun,
+			/obj/item/weapon/gun/grenadelauncher,
+		),
+		VERY_RARE_LOOT = list(
+			/obj/item/weapon/butterflyknife/viscerator,
+			/obj/item/weapon/c4,
+			/obj/item/weapon/caber/admin,
+			/obj/item/weapon/damocles,
+			/obj/item/weapon/grenade/spawnergrenade/bearnade,
+			/obj/item/weapon/grenade/spawnergrenade/beenade,
+			/obj/item/weapon/grenade/spawnergrenade/manhacks,
+			/obj/item/weapon/grenade/spawnergrenade/mothershipdrone,
+			/obj/item/weapon/grenade/spawnergrenade/spesscarp,
+			/obj/item/weapon/grenade/syndigrenade,
+			/obj/item/weapon/katana/hfrequency,
+			/obj/item/weapon/katana/magic,
+			/obj/item/weapon/organ_remover/traitor,
+			/obj/item/weapon/melee/morningstar/catechizer,
+			/obj/item/weapon/gun/banannon,
+			/obj/item/weapon/gun/bulletstorm,
+			/obj/item/weapon/gun/tesla/preloaded,
+			/obj/item/weapon/gun/stickybomb,
+			/obj/item/weapon/gun/osipr,
+			/obj/item/weapon/gun/grenadelauncher/syndicate,
+			/obj/item/weapon/gun/gatling,
+			/obj/item/weapon/gun/gatling/beegun,
+			/obj/item/weapon/gun/gatling/beegun/chillgun,
+			/obj/item/weapon/gun/gatling/beegun/hornetgun,
+		)
+	)
+
+/datum/loot_table/decoration
+	loot = list(
+		/obj/item/painting_brush,
+		/obj/item/candle,
+		/obj/item/weapon/reagent_containers/food/drinks/drinkingglass,
+		/obj/item/weapon/beach_ball,
+		/obj/item/trash/candle,
+		/obj/item/mounted/poster,
+		/obj/item/weapon/storage/photo_album,
+		/obj/item/device/flashlight/lamp,
+		/obj/item/device/flashlight/lamp/green,
+		/obj/item/weapon/reagent_containers/glass/beaker,
+		/obj/item/mounted/frame/painting,
+		/obj/item/mounted/frame/wreath,
+		/obj/item/weapon/reagent_containers/glass/rag,
+		/obj/item/weapon/lipstick/random,
+		/obj/item/weapon/reagent_containers/food/drinks/flask,
+		/obj/item/weapon/lighter/zippo,
+	)
+
+/datum/loot_table/engineering
+	loot = list(
+		/obj/item/tool/screwdriver,
+		/obj/item/tool/wrench,
+		/obj/item/tool/weldingtool,
+		/obj/item/tool/crowbar,
+		/obj/item/tool/wirecutters,
+		/obj/item/device/multitool,
+		/obj/item/device/t_scanner,
+		/obj/item/device/analyzer,
+		/obj/item/stack/cable_coil,
+		/obj/item/stack/rods,
+		/obj/item/device/flashlight,
+		/obj/item/taperoll/engineering,
+		/obj/item/weapon/extinguisher,
+		/obj/item/device/geiger_counter,
+		/obj/item/clothing/glasses/scanner/meson,
+		/obj/item/clothing/head/hardhat,
+		/obj/item/clothing/gloves/yellow,
+		/obj/item/weapon/rcl,
+		/obj/item/weapon/cell,
+		/obj/item/weapon/glowstick,
+		/obj/item/weapon/glowstick/red,
+		/obj/item/weapon/glowstick/blue,
+		/obj/item/weapon/circuitboard/airlock,
+		/obj/item/stack/sheet/metal,
+		/obj/item/stack/sheet/glass,
+		/obj/item/weapon/storage/belt/utility,
+	)
+
+/datum/loot_table/entertainment
```

Comment:
add the fumo variants, the bikehorn variants (skull and ankh) and the insulted gloves